From: Jeff Mahoney <jeffm@suse.com>
Subject: staging/sb105x: remove asm/segment.h dependency
Patch-mainline: Submitted to lkml/gregkh 3 Jan 2013

sb105x doesn't seem to actually need <asm/segment.h> (builds on x86
without it) and ppc/ppc64 doesn't provide it so it fails to build there.

This patch removes the dependency.

CC: Steven Rostedt <rostedt@goodmis.org>
Signed-off-by: Jeff Mahoney <jeffm@suse.com>
---

 drivers/staging/sb105x/sb_pci_mp.h |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/staging/sb105x/sb_pci_mp.h
+++ b/drivers/staging/sb105x/sb_pci_mp.h
@@ -19,7 +19,6 @@
 #include <asm/uaccess.h>
 #include <asm/io.h>
 #include <asm/irq.h>
-#include <asm/segment.h>
 #include <asm/serial.h>
 #include <linux/interrupt.h>
 

